package.json

```diff
@@ -65,7 +65,7 @@
     "eslint-plugin-jsx-a11y": "6.3.0",
     "eslint-plugin-react": "7.20.0",
     "eslint-plugin-react-hooks": "4.0.0",
-    "faker": "Marak/faker.js",
+    "faker": "thure/faker.js#v5.1.2",
```

Comment:
This PR also consumes my fork of Faker, which provides avatars served at `http://teams-admin.azurewebsites.net/assets/img/avatars`